mlir/lib/Dialect/Tosa/Transforms/TosaValidation.cpp

```diff
@@ -401,27 +406,39 @@ struct TosaValidation : public tosa::impl::TosaValidationBase<TosaValidation> {
 
     if (!profile.empty()) {
       for (std::string &prof : profile) {
-        auto profSymbol = symbolizeTosaProfileEnum(prof);
+        auto profSymbol = symbolizeProfile(prof);
         if (profSymbol) {
-          enabled_profiles.push_back(profSymbol.value());
+          targetEnv.addProfile(profSymbol.value());
+        } else {
+          llvm::errs() << "warning: unknown profile name passed in, supported "
+                          "profile are bi and mi\n";
```

Comment:
These should be updated to `pro_int` and `pro_fp`

Reply:
Done.